vue-components/nightwatch.config.js

```diff
@@ -1,8 +1,65 @@
+const build = `Nightwatch build-${process.env.DATE}`;
+
 /* eslint-disable @typescript-eslint/camelcase */
 module.exports = {
+	src_folders: [ 'tests/e2e/specs' ],
+	page_objects_path: 'tests/e2e/page-objects',
+	custom_commands_path: 'tests/e2e/custom-commands',
+
+	test_workers: {
+		enabled: true,
+		workers: 'auto',
+	},
+
 	test_settings: {
 		default: {
-			launch_url: 'http://localhost:6005/',
+			launch_url: 'https://wmde.github.io/wikit/vue-components/',
+		},
+		sauceLabs: {
+			selenium_host: `ondemand.${process.env.REGION === 'eu' ? 'eu-central-1.' : ''}saucelabs.com`,
+			selenium_port: 80,
+			username: process.env.SAUCE_USERNAME,
+			access_key: process.env.SAUCE_ACCESS_KEY,
```

Comment:
In order to run the tests on saucelabs one has to configure a username and access key locally as described in
https://wiki.saucelabs.com/display/DOCS/Best+Practice%3A+Use+Environment+Variables+for+Authentication+Credentials
The access key and username can be found at https://app.saucelabs.com/user-settings